src/views/Overview.tsx

```diff
@@ -1,19 +1,56 @@
-import { HStack } from '@chakra-ui/react'
+import { HStack, Input, useColorModeValue } from '@chakra-ui/react'
 import LeftPanel from '../components/overview/LeftPanel'
 import CollectionPanel from '../components/overview/CollectionPanel'
+import { useState } from 'react'
+
+export function SearchBar({
+  changeSearchValue
+}: {
+  changeSearchValue: (newValue: string) => void
+}) {
+  const cancelSearchButtonColour = useColorModeValue('#A0AEC0', '#52525b')
+  const borderColour = useColorModeValue(
+    'primary-light.500',
+    'primary-dark.500'
+  )
+
+  return (
+    <Input
+      type="search"
+      maxW={['full']}
```

Comment:
This is kinda redundant
```suggestion
      maxW="full"
```
would be equivalent I'm pretty sure